src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -344,6 +349,31 @@ class GoogleDataprocDAO(protected val dataprocConfig: DataprocConfig,
     executeGoogleRequestAsync(googleProject, "Bucket Path " + bucketPath.toUri, fileInserter).void
   }
 
+  /* set the pet service account as the staging bucket owner */
+  override def setStagingBucketOwnership(cluster: LeoCluster): Future[Unit] = {
+
+    // set owner access for the bucket itself
+    val bacInsert = {
+      val acl = new BucketAccessControl().setEntity(cluster.googleServiceAccount.value).setRole("OWNER")
```

Comment:
The entity string should be s"user-${cluster.googleServiceAccount.value}" (see the javadoc of `setEntity` -- this bit us before :)).

Reply:
blah.  thanks again.